crm/modules/amazon_payment/amazon_payment.inc.php

```diff
@@ -481,7 +484,7 @@ function amazon_payment_contact_delete_form ($cid) {
  * @param &$form_data Metadata about the form.
  * @param $form_id The name of the form.
  */
-function amazon_payment_form_alter($form, $form_id) {
+function amazon_payment_form_alter ($form, $form_id) {
```

Comment:
just comparing it to other functions in the same file that seem to have spaces before & after it